### ISSUE TITLE
Tests Blitter/bltint/bltint2 to bltint6 fail

### ISSUE THREAD

**Issue by dirkwhoffmann** [dirkwhoffmann/vAmiga#549]
These tests are similar to bltint1 with the CLR command replaced by a series of NOPs.

Amiga 500 8A (bltint2): 🥰

![bltint2_A500_8A](https://user-images.githubusercontent.com/12561945/135124958-ccc9d648-1332-48ff-a38d-986c3fcb01d3.JPG)

vAmiga (bltint2): 🙈

<img width="886" alt="Bildschirmfoto 2021-09-28 um 18 08 55" src="https://user-images.githubusercontent.com/12561945/135125314-f0bb36e5-f6f6-4c07-9983-40001dfa9493.png">

**Comment by dirkwhoffmann** [comment]
Fixed. The Blitter interrupt was signalled one DMA cycle too early.

Old code:

```
    // Trigger Blitter interrupt if this is the termination cycle
    if ((instr & BLTDONE) && !birq) {
        signalEnd();
        paula.raiseIrq(INT_BLIT);
        birq = true;
    }
```

New code:

```
   // Trigger Blitter interrupt if this is the termination cycle
    if ((instr & BLTDONE) && !birq) {
        signalEnd();
        paula.scheduleIrqRel(INT_BLIT, DMA_CYCLES(1));
        birq = true;
    }
```

<img width="886" alt="Bildschirmfoto 2021-09-29 um 12 18 08" src="https://user-images.githubusercontent.com/12561945/135250707-721671bb-32c8-4bcc-8622-976d69ec90b1.png">

Interestingly, the code change doesn't fix #548 (bltint1) which means that the original issues #466 is actually caused by two independent bugs:
- A Blitter bug related to interrupt timing (fixed)
- A CPU bug related to IPL polling inside the CLR instruction (not fixed yet)

**Comment by dirkwhoffmann** [comment]
Fixed in vAmiga 1.1b4.
